### PR TITLE
Fix shadows on Android

### PR DESCRIPTION
**Pull request by osdnk**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -309,7 +310,7 @@
         "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
         "type": "android.emulator",
         "device": {
-          "avdName": "Nexus_5X_API_29_x86"
+          "avdName": "Nexus_5X_API_29rr_x86"
```
Comment (brunobar79) [inline package.json:313]: Can u revert this one?

**Review by brunobar79 — APPROVED** [review]
💯

File: package.json (hunk 1 above)
Comment (osdnk) [reply package.json:313]: rr

Comment (osdnk) [inline package.json:313]:
```suggestion
          "avdName": "Nexus_5X_API_29_x86"
```

**Comment by osdnk** [comment]
TODO later: remove it from iOS - there's no need for this package on iOS

**Pull request merged**